core-java-modules/core-java-numbers-7/src/test/java/com/baeldung/convertphonenumberinwordstonumber/UseHashMapToConvertPhoneNumberInWordToNumberUnitTest.java

```diff
@@ -0,0 +1,51 @@
+import org.junit.jupiter.api.Test;
+import java.util.Map;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class UseHashMapToConvertPhoneNumberInWordToNumberUnitTest {
```

Comment:
JUnit 5 classes don't need `public` on the `class` or the methods

Reply:
Done.